tools/languagelocalizer/languageLocalizer.js

```diff
@@ -362,7 +362,7 @@ const LanguageLocalizer = {
 
       const cldrLanguages =
           (await this.retrieveJson(
-               `https://raw.githubusercontent.com/unicode-org/cldr-json/main/cldr-json/cldr-localenames-modern/main/${
+               `https://raw.githubusercontent.com/unicode-org/cldr-json/main/cldr-json/cldr-localenames-full/main/${
```

Comment:
Curious about this change. Do we need a comment here about use `*-full` for future us.

Reply:
I found that there were slightly more languages available in the `*-full` repository, and this resulted in a slightly more complete language database. This is why we gained some extra translations and currencies for Interlingua and Corsican in this PR.